stdlib/3.4/asyncio/locks.pyi

```diff
@@ -49,11 +49,11 @@ class Condition(_ContextManagerMixin):
     def notify_all(self) -> None: ...
 
 class Semaphore(_ContextManagerMixin):
-    def __init__(self, value: int = 1, *, loop: AbstractEventLoop = None) -> None: ...
+    def __init__(self, value: int = ..., *, loop: AbstractEventLoop = ...) -> None: ...
```

Comment:
But having `= None` implicitly modifies the type to become `Optional[]` so either keep the `None` or use `Optional[AbstractEventLoop]`.